modules/btcpayserver.nix

```
@@ -84,6 +84,12 @@ in {
         default = null;
         description = "The lightning node implementation to use.";
       };
+      rootpath = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "btcpayserver";
+        description = "The root path in the URL to access btcpayserver.";
```

```suggestion
        description = "The prefix for root-relative btcpayserver URLs.";
```
Sorry for bringing this up again, but now we have a really clear description.

---

No problem, I care about having accurate documentation.

Fixed in edc657d138e51970e0ea981fbbc2d3471d0cda41